docs/contributing/05-gh-actions.md

```diff
@@ -2,205 +2,118 @@
 
 - [GitHub Actions](#github-actions)
   - [Introduction](#introduction)
-  - [Restrictions of use](#restrictions-of-use)
+  - [Restrictions](#restrictions)
+  - [Allowed Github Actions](#allowed-github-actions)
   - [GitHub Actions Security Best Practices](#github-actions-security-best-practices)
-  - [Appendix A: Approved GitHub Actions](#appendix-a-approved-github-actions)
-    - [golangci/golangci-lint-action](#golangcigolangci-lint-action)
-    - [AbsaOSS/k3d-action](#absaossk3d-action)
-    - [zgosalvez/github-actions-ensure-sha-pinned-actions](#zgosalvezgithub-actions-ensure-sha-pinned-actions)
-    - [ahmadnassri/action-workflow-queue](#ahmadnassriaction-workflow-queue)
-    - [autotelic/action-wait-for-status-check](#autotelicaction-wait-for-status-check)
-    - [gaurav-nelson/github-action-markdown-link-check](#gaurav-nelsongithub-action-markdown-link-check)
-    - [slackapi/slack-github-action](#slackapislack-github-action)
-  - [Not allowed GitHub Actions](#not-allowed-gh-actions)
-    - [shmokmt/actions-setup-github-comment](#shmokmtactions-setup-github-comment)
-    - [shmokmt/actions-setup-tfcmt](#shmokmtactions-setup-tfcmt)
-    - [CondeNast/conventional-pull-request-action](#condenastconventional-pull-request-action)
+    - [Security review](#security-review)
+    - [Set the minimal scope for credentials](#set-the-minimal-scope-for-credentials)
+    - [Do NOT use `pull_request_target` event](#do-not-use-pull_request_target-event)
+    - [Treat event context data as untrusted input](#treat-event-context-data-as-untrusted-input)
+    - [Use Dependabot](#use-dependabot)
+    - [Review Action changes](#review-action-changes)
+  - [Additional information](#additional-information)
 
 ## Introduction
 
 GitHub Actions are a powerful tool to automate almost every task of the development cycle. The ecosystem of GitHub Actions is growing very fast and one can find a prepared Action for a lot of activities. But like all other usages of Open Source Software, GitHub Actions must be handled carefully because an action is running external code on our code in our GitHub repositories.
```

Comment:
I find using lowercase and uppercase "A" for the noun "actions" inconsistent. I guess "Actions" should be capitalized when referring to the feature and not capitalized when talking about a single GitHub action. WDYT?